### PR TITLE
LPS-147477 update Congrats Actions Fragment to Mobile and Tablet View

### PR DESCRIPTION
**Pull request by Mylena-Rodrigues**

### CODE REVIEW

**Comment by Mylena-Rodrigues** [comment]
ci:test:sf

**Comment by Mylena-Rodrigues** [comment]
ci:test:sf

**Comment by Mylena-Rodrigues** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-raylife/src/main/resources/site-initializer/fragments/raylife/fragments/congrats-actions/index.css [hunk 1]
```diff
@@ -1,5 +1,10 @@
...
-	display: flex;
-	justify-content: center;
-	margin-top: 10px;
+@media screen and (max-width: 767.98px){
+	.congrats-actions{		
+		align-items: center;
+		flex-direction: column;
+	}
+	
+	.congrats-actions button{
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-raylife/src/main/resources/site-initializer/fragments/raylife/fragments/congrats-actions/index.css:7]:
```suggestion
	.congrats-actions button {
```

File: modules/apps/site-initializer/site-initializer-raylife/src/main/resources/site-initializer/fragments/raylife/fragments/congrats-actions/index.css [hunk 2]
```diff
@@ -1,5 +1,10 @@
-.congrats-actions {
-	display: flex;
-	justify-content: center;
-	margin-top: 10px;
+@media screen and (max-width: 767.98px){
+	.congrats-actions{
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-raylife/src/main/resources/site-initializer/fragments/raylife/fragments/congrats-actions/index.css:2]:
```suggestion
	.congrats-actions {		
```

File: modules/apps/site-initializer/site-initializer-raylife/src/main/resources/site-initializer/fragments/raylife/fragments/congrats-actions/index.css [hunk 3]
```diff
@@ -1,5 +1,10 @@
-.congrats-actions {
-	display: flex;
-	justify-content: center;
-	margin-top: 10px;
+@media screen and (max-width: 767.98px){
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-raylife/src/main/resources/site-initializer/fragments/raylife/fragments/congrats-actions/index.css:1]:
```suggestion
@media screen and (max-width: 767.98px) {
```

**Review by kevenleone — CHANGES_REQUESTED** [review]
@Mylena-Rodrigues PR looks good, just format sourcing the CSS file

**Comment by Mylena-Rodrigues** [comment]
@kevenleone, I uploaded the requested changes, please, check if is ok

**Comment by kevenleone** [comment]
ci:forward

**Review by kevenleone — APPROVED** [review]
LGTM 🔥 

Thanks @Mylena-Rodrigues

**Comment by Mylena-Rodrigues** [comment]
ci:stop

**Comment by Mylena-Rodrigues** [comment]
ci:forward

**Pull request closed without merging**